index.html

```diff
@@ -181,7 +178,7 @@ <h2 class="subtitle">Project names goes here</h2>
                         <!-- See this project button -->
                         <div class="box align-button box6">
                             <a href="#">
-                                <div class="buttom-arrow">
+                                <div class="buttom-arrow" onclick="popUp('project4')">
                                     <p>See this project
                                     <img class="arrow" src="/assets/211621_c_right_arrow_icon.png" alt="">
                                     </p>
```

Comment:
- [ ] You have done a very good job rendering your projects dynamically from javascript. Kudos to that👍 

Please kindly ensure you remove all the hardcoded projects lists you have on your html file i.e. All the list of projects you added in your HTML should be removed since you are already displaying the projects dynamically from javascript 🤓 

****N:B** All projects are to be displayed from javascript and not in the HTML file.**

Reply:
### Hello Vickymarz, thank you very much for your comments. @vickymarz 

I do not understand the request, if you have a moment, I would appreciate it if you would give me a few minutes on Zoom Call so that I can fully understand what is left over in my project.

![image](https://user-images.githubusercontent.com/106561762/180106925-319fd560-b974-44d9-83b0-eccddf25814e.png)

Everything about the popUp menu is in the JavaScript file. That which appears in your comment, the function that is activated when clicking on the "See this project" button. 

Could you explain in more detail what I have to remove from my template please? Beforehand thank you very much

Reply:
We can join a zoom call so I can put you through

Reply:
### Thank you very much! 
Thank you very much for all your support. In the end, if I managed to remove all the content of the projects and store them in the JavaScript file. I had to leave the <div> out of the sections because you used them to structure the Grid in the desktop version.